src/protagonist/DLCS.Repository/NamedQueries/Parsing/BaseNamedQueryParser.cs

```diff
@@ -123,15 +124,15 @@ private T GenerateParsedNamedQuery(int customerId, string[] templatePairing, Lis
                         break;
                     case Number1:
                         assetQuery.Number1 =
-                            long.Parse(GetQueryArgumentFromTemplateElement(queryArgs, elements[1]));
+                            ConvertIntegerQueryArg(GetQueryArgumentFromTemplateElement(queryArgs, elements[1]));

```

Comment:
Behaviour change - was `long.Parse` but it's now `int.Parse`

Reply:
could set the return from `ConvertIntegerQueryArg` to `long`?